### PR TITLE
Automation

### PR DESCRIPTION
**Pull request by saadrahim**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The AMD GPU Operator simplifies the deployment and management of AMD Instinct GP
 ## Compatibility
 
 - **Kubernetes**: 1.29.0
-- Please refer to the [ROCM documentaiton](https://rocm.docs.amd.com/en/latest/compatibility/compatibility-matrix.html) for the compatability matrix for the AMD GPU DKMS driver.
+- Please refer to the [ROCm documentaiton](https://rocm.docs.amd.com/en/latest/compatibility/compatibility-matrix.html) for the compatability matrix for the AMD GPU DKMS driver.
```
Comment (farshadghodsian) [inline docs/index.md:17]: Documentation is misspelled here.

Comment (saadrahim) [reply docs/index.md:17]: I only fixed ROCm, I will fix the other. But this PR add linting for spell check. Can you turn the github actions?

Comment (saadrahim) [reply docs/index.md]: Maybe github actions are on and this PR just needs to be merged. It probably doesn't run github action updates from a fork.

File: docs/conf.py [hunk 2]
```diff
@@ -14,12 +14,18 @@
...
 html_theme = "rocm_docs_theme"
+main_doc_link = {
+    "Instinct&#8482 Hub": "https://dcgpu.docs.amd.com"
+},
 html_theme_options = {
     "flavor": "generic",
-    "header_title": "AMD Instinct Documentation",
+    "header_title": "Instinct&#8482 Hub",
+    "header_link": "https://dcgpu.docs.amd.com",
+    "main_doc_link": ("Instinct&#8482 Hub", "https://dcgpu.docs.amd.com"),
```
Comment (saadrahim) [inline docs/conf.py:24]:
```suggestion
    "link_main_doc": ("Instinct&#8482 Hub", "https://dcgpu.docs.amd.com"),
```

**Comment by saadrahim** [comment]
Please squash merge this PR.

**Pull request merged**